drivers/sensor/sensirion/scd4x/scd4x.c

```diff
@@ -166,96 +223,21 @@ static int scd4x_init(const struct device *dev)
 
     return 0;
 }
-/** Read buffer */
-static char read_buf[3 * MAX_READ_SIZE];
 
```

Comment:
I don't know why this code is here so, I don't know if it will be used in the futur. Keeping commented to let the creator retreive it if needed. 

```suggestion
/*
static int scd4x_send_and_fetch(const struct device *dev, uint16_t cmd, uint16_t *val)
{
    int ret;

    ret = scd4x_send_command(dev, cmd);
    if (ret < 0) {
        return ret;
    }

    k_sleep(K_MSEC(1));

    return scd4x_read(dev, cmd, 1, val);
*/
}
```